docs/development/product-roadmap.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Product Roadmap
+---
+
+# Product Roadmap
+
+## You are here
+
+The Koor Storage Distribution includes features that work on top of Rook and Ceph to enhance the experience of storage professionals. Today, the KSD functionality includes the following:
+
+* Koor Storage Distribution (KSD)
+  * Bundles latest stable release of Rook with additional tools from Koor
+  * A KSD license includes authorization to use everything in this list
+* Koor Operator
+  * Installs KSD on any Kubernetes cluster, and sets up Rook Ceph storage
+  * Uses Helm chart or CRDs to define storage
+  * Checks the status of storage
+* Diagnostic scripts
+  * Gathers debugging information that is useful for troubleshooting issues in a Rook Ceph cluster
+* Extended Ceph Exporter
+  * Extends metrics exported from Promethius for improved utilization charts
+
+### Releases
+
+* [Koor Operator v0.2.0](https://github.com/koor-tech/koor-operator/releases/tag/koor-operator-0.2.0)
+* [Extended Ceph Exporter v1.2.8](https://github.com/koor-tech/extended-ceph-exporter/releases/tag/extended-ceph-exporter-1.2.8)
+* [Koor gather info scripts](https://github.com/koor-tech/koor-gather-info)
```

Comment:
We should probably start versioning this

Reply:
agreed